src/cluster_legacy.c

```diff
@@ -3196,7 +3196,7 @@ int clusterProcessPacket(clusterLink *link) {
                 if (bitmapTestBit(hdr->myslots, j)) {
                     if (server.cluster->slots[j] == sender || isSlotUnclaimed(j)) continue;
                     if (server.cluster->slots[j]->configEpoch > senderConfigEpoch) {
-                        serverLog(LL_VERBOSE,
+                        serverLog(LL_NOTICE,
```

Comment:
I don't think this is a NOTICE and would leave it at VERBOSE. A lot of these will get bounced around during slot migration, and I think it's mostly just noise for people looking through the log.